packages/uniforms-antd/src/ListField.tsx

```diff
@@ -48,13 +48,17 @@ function List({
   wrapperCol,
   ...props
 }: ListFieldProps) {
+  const wrapperStyle = error
+    ? { borderColor: 'rgb(255, 85, 0)', ...style }
```

Comment:
Is it worth setting this error style to const and spread it here or just leave it, because its usage is rare?

Reply:
Yes, great idea. Something like this:
```ts
const wrapperStyle = error
  ? style ? { ...errorStyle, ...style } : errorStyle
  : style;
```